### PR TITLE
chore: lab_6-7 update

### PR DESCRIPTION
**Pull request by Szymon-Budziak**

### CODE REVIEW

File: lab_6-7/lab6-7-index-opt-3.md [hunk 1]
```diff
@@ -221,33 +233,35 @@ where orderdate between '2010-10-01' and '2011-06-01'
...
 
-Plan dla zapytania 5:
+_Execution Plan dla zapytania 5_
 
 ![](img/ex2/plan5.png)
 
-> Plany wykonania wszystkich trzech zapytań wyglądają tak samo.
+> Wyniki dwóch pierwszych zapytań wyglądają tak samo. Przy wykonywaniu zapytania pierwszego, - bez spacyfikowania kolejności, - dostajemy taki sam wynik jak przy kolejności rosnącej. Oznacza to, że właśnie w takiej kolejności posortowany jest indeks.
+
+> Możemy zauważyć, że w drugim zapytaniu, że indeks został wykorzystany, na co wskazuje operacja `Clustered Index Seek` w _Execution Plan dla zapytania 4_, jednak żadne dodatkowe sortowanie nie zostało wykonane. Jeśli chodzi o trzeci zapytanie, to również widzimy, że indeks został wykorzystany. Wskazuje na to operacja `Clustered Index Seek` w _Execution Plan dla zapytania 5_. Nie jest wykonywane dodatkowe sortowanie. Plany wykonania wszystkich trzech zapytań wyglądają tak samo.
```
Comment (my-alaska) [inline lab_6-7/lab6-7-index-opt-3.md:262]: powtórzone "że"

File: lab_6-7/lab6-7-index-opt-3.md [hunk 2]
```diff
@@ -591,19 +631,25 @@ GROUP BY Name, Surname
...
 | 2         | ![](img/ex4/query6plan.png) |
-| 3         | ![](img/ex4/query7plan.png) |
+
+**Rekomendacja Databse Engine Tuning Advisor**
+
+![](img/ex4/recommend3.png)
+
+> Narzędzie sugeruje utworzeniu indeksu, który został przez nas już zdefiniowany, czyli indeks wykorzystujący kilka atrybutów.
+
+> Wniosek: Indeksy wykorzystujące kilka atrybutów, wraz z include są przydatne, gdy zapytania obejmują wiele kolumn w warunkach wyszukiwania zapytania.
```
Comment (my-alaska) [inline lab_6-7/lab6-7-index-opt-3.md:648]: powtórzenie - "zapytania [...] w warunkach zapytania"

File: lab_6-7/lab6-7-index-opt-3.md (hunk 1 above)
Comment (Szymon-Budziak) [reply lab_6-7/lab6-7-index-opt-3.md:262]: deleted repeated "że"

File: lab_6-7/lab6-7-index-opt-3.md (hunk 2 above)
Comment (Szymon-Budziak) [reply lab_6-7/lab6-7-index-opt-3.md:648]: Refactored this text

**Pull request merged**